### PR TITLE
Lvs

### PR DESCRIPTION
**Pull request by zbintliff**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### New Plugins

- [x] Tests

- [x] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by zbintliff** [comment]
As the world moves towards docker and ECS we needed a way to monitor our thinpool. This is how we are monitoring the actual data usage percent of the logical volumes.

File: bin/check-lv-usage.rb [hunk 1]
```diff
@@ -72,7 +72,7 @@ class CheckLVUsage < Sensu::Plugin::Check::CLI
          default: 95
   # Setup variables
   #
-  def initialize
+  def initialize(argv = ARGV)
     super
```
Comment (insom) [inline bin/check-lv-usage.rb]: This shouldn't be needed. `Sensu::Plugin::CLI` (the base class which uses `argv`) already has the default of `argv=ARGV` -- you're not using this variable yourself so I think this just looks strange.

File: Rakefile [hunk 2]
```diff
@@ -29,7 +23,7 @@ end
 
 desc 'Make all plugins executable'
 task :make_bin_executable do
-  `chmod -R +x bin/*.rb`
+  `chmod -R +x bin/*`
```
Comment (insom) [inline Rakefile]: Sensu checks are supposed to use file extensions, for Windows compatability. (Yes, I know no one will be able to run the LVM plugin on Windows but :shrug:).

File: Rakefile [hunk 3]
```diff
@@ -5,15 +5,9 @@ require 'rspec/core/rake_task'
...
 
-desc 'Don\'t run Rubocop for unsupported versions'
-begin
-  args = if RUBY_VERSION >= '2.0.0'
-           [:spec, :make_bin_executable, :yard, :rubocop, :check_binstubs]
-         else
-           [:spec, :make_bin_executable, :yard]
-         end
-end
+args = [:spec, :make_bin_executable, :yard, :rubocop, :check_binstubs]
```
Comment (insom) [inline Rakefile]: This is good, I believe Ruby 1.9 support was dropped so I don't need this in the gemspec. Thanks.

File: sensu-plugins-lvm.gemspec [hunk 4]
```diff
@@ -2,12 +2,7 @@ lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 require 'date'
-
-if RUBY_VERSION < '2.0.0'
-  require 'sensu-plugins-lvm'
-else
-  require_relative 'lib/sensu-plugins-lvm'
-end
+require_relative 'lib/sensu-plugins-lvm'
```
Comment (insom) [inline sensu-plugins-lvm.gemspec]: :+1:

File: bin/check-lv-usage.rb (hunk 1 above)
Comment (zbintliff) [inline bin/check-lv-usage.rb]: If you want I can pass `argv` down to `super`?

**Pull request merged**